packages/sitecore-jss/src/personalize/graphql-personalize-service.ts

```diff
@@ -97,7 +97,7 @@ export class GraphQLPersonalizeService {
       return data?.layout?.item
         ? {
             // CDP expects content id format `<id>_<language>_<version>` (lowercase)
-            contentId: `${data.layout.item.id}_${language}_${data.layout.item.version}`.toLowerCase(),
+            contentId: `embedded_${data.layout.item.id}_${language}`.toLowerCase(),
```

Comment:
According to the spec - on language, we also need to replace dashes(-) with underscores(_). We should add a unit test for this case too.